### PR TITLE
Feat/ccn3 943/your personal details

### PR DESCRIPTION
**Pull request by kingainformed**
## Description

Personal details task.

## Todo's before PR

- [x] Branch has been rebased against develop
- [x] Unit tests passed (`make test`)
- [x] PR named appropriately - see [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [x] PR description describes the overall goals of PR commits
- [x] Templates have been checked against the relevant user-story

## Migrations

- [ ] Null fields in models specifies default value
- [ ] You generated and applied migrations (`make migrate`)
- [ ] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [x] Specified reviewers (even if it's yourself)
- [x] Specified assignees (those who'll merge)
- [x] Specified labels

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [x] Code syntax formatting checked
- [x] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [x] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

File: personal_details_app/templates/certificates_of_good_conduct.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+{{ form.error_summary }}
+
+
+<h1 class="form-title heading-large">Certificates of good conduct</h1>
+
+<form method="post" novalidate {% if form.is_multipart %}enctype="multipart/form-data" {% endif %}>
+
+    {% csrf_token %}
+    <p>You need to have certificates of good conduct from your time living abroad in the last 5 years.</p>
+    <p>Certificated of good conduct prove that you don't have a criminal record.</p>
```
Comment (ricaduchateau) [inline personal_details_app/templates/certificates_of_good_conduct.html]: Should be 'certificates'.

File: personal_details_app/templates/certificates_of_good_conduct.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <p>Contact the countries' embassies to get original certificates.</p>
+
+    <a href="{{ link_url }}" class="button">Continue</a>
+</form>
+<br>
+<a href="{% url 'Task-List' %}?id={{ id }}">Return to list</a>
```
Comment (ricaduchateau) [inline personal_details_app/templates/certificates_of_good_conduct.html]: Can you use the include return to list line here?

File: personal_details_app/templates/address-details.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+<form method="post" novalidate {% if form.is_multipart %}enctype="multipart/form-data" {% endif %}>
+
+    {% csrf_token %}
+    <p> {{ address }}</p>
+
+    <input type="submit" name="action" class="button" value="Save and continue">
+</form>
+<br>
+<a href="{% url 'Task-List' %}?id={{ id }}">Return to list</a>
```
Comment (ricaduchateau) [inline personal_details_app/templates/address-details.html]: Can you use the include return to list line here?

File: personal_details_app/templates/dob.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+<form method="post" novalidate {% if form.is_multipart %}enctype="multipart/form-data" {% endif %}>
+
+    {% csrf_token %}
+    {{ form }}
+
+    <input type="submit" name="action" class="button" value="Save and continue">
+</form>
+<br>
+<a href="{% url 'Task-List' %}?id={{ id }}">Return to list</a>
```
Comment (ricaduchateau) [inline personal_details_app/templates/dob.html]: Can you use the include return to list line here?

File: personal_details_app/templates/lived_abroad.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+<form method="post" novalidate {% if form.is_multipart %}enctype="multipart/form-data" {% endif %}>
+
+    {% csrf_token %}
+    {{ form }}
+
+    <input type="submit" name="action" class="button" value="Save and continue">
+</form>
+<br>
+<a href="{% url 'Task-List' %}?id={{ id }}">Return to list</a>
```
Comment (ricaduchateau) [inline personal_details_app/templates/lived_abroad.html]: Can you use the include return to list line here?

File: personal_details_app/templates/names.html [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    <p>Please give us your full name, including any middle names</p>
+    <p>The details must match what is in your criminal record (DBS) check</p>
+
+    {% csrf_token %}
+    {{ form }}
+
+    <input type="submit" name="action" class="button" value="Save and continue">
+</form>
+<br>
+<a href="{% url 'Task-List' %}?id={{ id }}">Return to list</a>
```
Comment (ricaduchateau) [inline personal_details_app/templates/names.html]: Can you use the include return to list line here?

File: personal_details_app/templates/send_certificates.html [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+<form method="post" novalidate {% if form.is_multipart %}enctype="multipart/form-data" {% endif %}>
+
+    {% csrf_token %}
+    <p>You must post your original certificates (not photocopies) to Ofsted.</p>
+    <p>We will check your certificates and send it back to you.</p>
+
+    <a href="{{ link_url }}" class="button">Save and continue</a>
+</form>
+<br>
+<a href="{% url 'Task-List' %}?id={{ id }}">Return to list</a>
```
Comment (ricaduchateau) [inline personal_details_app/templates/send_certificates.html]: Can you use the include return to list line here?

**Comment by kingainformed** [comment]
Hi @ricaduchateau, these changes have been made.

**Pull request merged**